### PR TITLE
Add iris_rtps model

### PR DESCRIPTION
**Pull request by stmoon**
To run multiple irises based on RTPS, I add the iris_rtps model.

please check the related PR of firmware (https://github.com/PX4/Firmware/pull/15463)

### CODE REVIEW

**Comment by TSC21** [comment]
Why is this needed? We already have https://github.com/PX4/sitl_gazebo/tree/master/models/iris_rtps

File: models/rotors_description/urdf/iris_rtps.xacro [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+  - ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  - POSSIBILITY OF SUCH DAMAGE.
+  -
+  -->
+
+<robot name="iris_rtps" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- Included URDF Files -->
+  <xacro:include filename="$(arg rotors_description_dir)/urdf/iris.xacro" />
+</robot>
```
Comment (Jaeyoung-Lim) [inline models/rotors_description/urdf/iris_rtps.xacro]: This file is not being used

**Review by Jaeyoung-Lim — COMMENTED** [review]
@TSC21 I asked @stmoon to create this since the multivehicle script relies on the xacro macro.

**Comment by TSC21** [comment]
> @TSC21 I asked @stmoon to create this since the multivehicle script relies on the xacro macro.

Is there a way of not having it dependent? It seems to be a waste.

**Comment by Jaeyoung-Lim** [comment]
@TSC21 Yes, I think we need to definitely fix this, but to enable https://github.com/PX4/Firmware/pull/15463 I thought it is sufficient, given that we are using separate xacro files for all the models we support.

Would be great to discuss on how we can fix this - recently there are more and more people running multivehicle simulations with various configurations (e.g. gimbals / camera sensors) and the dependency to xacro is a blocker on supporting such use cases.

**Comment by Jaeyoung-Lim** [comment]
@stmoon Any updates?

**Comment by stmoon** [comment]
> @stmoon Any updates?

Actually, I'm waiting for the decision. If PX4 decide the concept, I will follow it.

**Comment by TSC21** [comment]
Let's bring this in then.

File: models/rotors_description/urdf/iris_rtps.xacro (hunk 1 above)
Comment (TSC21) [reply models/rotors_description/urdf/iris_rtps.xacro]: @stmoon please fix this.

Comment (stmoon) [reply models/rotors_description/urdf/iris_rtps.xacro]: I remove it.

**Review by Jaeyoung-Lim — APPROVED** [review]

**Pull request merged**